### DISCUSSION TITLE
[求助] 新手绳匠如何快速提升绳网等级？

### DISCUSSION THREAD

**Discussion by ggdwj233** [share121/inter-knot#69] [其他]
如题~刚当上绳匠没多久，在绳网上看到好多人和我一样都是1级，大家都是新手吗？但是，偶尔会看到绳网等级超高的人，有什么快速升级的技巧吗？
![958FC610B4D3634AE12105CE80CA0A7F](https://github.com/user-attachments/assets/1aa7eb80-d49a-4fd4-b7de-71bbdf19d42e)

**Comment by RS64671012** [comment]
可能还没有开发，暂时不清楚

**Comment by share121** [comment]
绳网等级是根据 Github 的公开仓库数量来判定

**Reply by MetsukiMio** [reply]
比如我（
![屏幕截图_20240809_164919-1](https://github.com/user-attachments/assets/2259c209-52b7-470c-9eba-8e01bcdbb2ee)
![屏幕截图_20240809_165018](https://github.com/user-attachments/assets/47944b6a-6db8-4664-8102-6041532db988)

**Comment by McQueen979** [comment]
经验+3

**Comment by RS64671012** [comment]
顺便问一句，怎么看别人的等级

**Comment by zhuxuepengcha** [comment]
要多接委托呀，但是新手期不要接太难的，容易出问题
